README.md

```diff
@@ -17,6 +17,13 @@ Do you have an idea for pushing the Filecoin ecosystem forward? Grants up to $50
 
 ---
 
+### Documentation Enhancement Grant
+Do you have a great idea to enhance existing Filecoin documentation or to create new resources? Consider applying for a Documentation Enhancement Grant! Grants up to $5,000 are available to those interested in contributing to the enhancement of Filecoin documentation.
+
+[**LEARN MORE ABOUT OPEN GRANTS**](LINK HERE) **AND** [**APPLY FOR AN OPEN GRANT**](LINK HERE)
```

Comment:
Update from Open Grant to Documentation Enhancement Grant and it should be good.

Reply:
Great catch!